master/static/migrations/20220228132402_store-profile-pics.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE user_profile_images (
+  id SERIAL PRIMARY KEY,
+  username TEXT NOT NULL,
```

Comment:
can it be a foreign key on user id instead of username? 

>One feature request was that when a user is deleted, their image is deleted. The table is not yet linked strongly enough to the users table.

Then we could use `ON DELETE CASCADE` to delete these automatically: https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK

Reply:
:+1: agreed, I can do this now!  I had been querying this table username, but now that the query looks up the users table and does a join, I can replace with user_id. Just made a commit.